Makefile.frag

```diff
@@ -137,11 +141,13 @@ docs:
 release-docs: docs
 	mkdocs gh-deploy --clean
 
-package.xml:
+package.xml: RELEASE
 	php bin/prep-release.php $(MONGODB_VERSION)-$(MONGODB_STABILITY)
 
 RELEASE:
-	@git log --pretty=format:"%ad  %an  <%ae>%n%x09* %s%n" --date short --since="$$(git show -s --format=%ad `git rev-list --tags --max-count=1`)" > RELEASE-$(MONGODB_VERSION)
+	echo "RELEASE $(MONGODB_VERSION)" >> RELEASE-$(MONGODB_VERSION)
+	echo "-------------" >> RELEASE-$(MONGODB_VERSION)
+	@git log --pretty=format:"%ad  %an  <%ae>%n%x09* %s%n" --date short --since="$$(git show -s --format=%ad `git rev-list --tags --max-count=1`)" >> RELEASE-$(MONGODB_VERSION)
```

Comment:
This currently generates a new release file with the full version. Is that intentional, as you only join it during packaging?